app/src/editor/canvas/components/ModuleSearch.jsx

```diff
@@ -475,12 +475,12 @@ export class ModuleSearch extends React.PureComponent<Props, State> {
                                 <div className={cx(styles.Header, styles.dragHandle)}>
                                     <button type="button" className={styles.minimize} onClick={() => this.toggleMinimize()}>
                                         {isExpanded ?
-                                            <SvgIcon name="caretUp" /> :
-                                            <SvgIcon name="caretDown" />
+                                            <SvgIcon name="brevetDown" className={styles.flip} /> :
+                                            <SvgIcon name="brevetDown" className={styles.normal} />
```

Comment:
👍

Reply:
I wonder if this would be worthy of making a part of SvgIcon itself

Reply:
Yeah, it would be nice! Created https://streamr.atlassian.net/browse/PLATFORM-596